css/styles.css

```diff
@@ -512,6 +551,74 @@ a.btn-connect {
   .work-section .project:nth-child(2n+1) .details {
     background-color: transparent;
   }
+
+  /* The Modal (background) */
```

Comment:
- [ ] OPTIONAL
To enhance a cleaner code kindly endeavor to remove **all** comments within your hosted code